### ISSUE TITLE
How to change leverage and its mode on MECX?

### ISSUE THREAD

**Issue by KindSpidey** [ccxt/ccxt#13256]
How i can change leverages and set mode for futures on MECX with ccxt?

```
mexc_futures = ccxt.mexc({
    "apiKey": config.MEXC_API_KEY,
    "secret": config.MEXC_SECRET_KEY,
    "options": {'defaultType': 'swap' }
})
mexc_futures.set_leverage(10, "BTC/USDT")
```
Got
```
ccxt.base.errors.ArgumentsRequired: mexc setLeverage() requires a positionId parameter
```

**Comment by kroitor** [comment]
- https://github.com/ccxt/ccxt/wiki/Manual#overriding-unified-api-params

```Python
from pprint import pprint
mexc_futures = ccxt.mexc({
    "apiKey": config.MEXC_API_KEY,
    "secret": config.MEXC_SECRET_KEY,
    "options": {'defaultType': 'swap' }
})
positions = mexc_futures.fetch_positions()
pprint(positions)
mexc_futures.set_leverage(10, "BTC/USDT", {'positionId': YOUR_POSITION_ID})
```

**Comment by KindSpidey** [comment]
Well, it is for already opened postions, but i need code for creating position with exact leverage and its mode

**Comment by kroitor** [comment]
@KindSpidley please, upgrade to CCXT 1.82.54 (the new build will arrive in 20 minutes) and use this code:

```Python
from pprint import pprint
print('CCXT Version:', ccxt.__version__)
mexc_futures = ccxt.mexc({
    "apiKey": config.MEXC_API_KEY,
    "secret": config.MEXC_SECRET_KEY,
    "options": {'defaultType': 'swap' }
})
response = mexc_futures.set_leverage(10, "BTC/USDT", {
    'openType': 1, # isolated, 2 = cross
    'positionType': 1, # 1 = long, 2 = short
})
pprint(response)
```

**Comment by KindSpidey** [comment]
Shows only 53 version
![изображение_2022-05-15_162125375](https://user-images.githubusercontent.com/44064808/168470173-4835375d-57e7-41de-b863-59bd6b036166.png)

**Comment by kroitor** [comment]
@KindSpidey it is 1.82.58 at this point.

**Comment by KindSpidey** [comment]
This
```
mexc_futures = ccxt.mexc({
    "apiKey": config.MEXC_API_KEY,
    "secret": config.MEXC_SECRET_KEY,
    "options": {'defaultType': 'swap' }
})
balance = mexc_futures.fetch_balance()["USDT"]["free"]
mexc_futures.set_leverage(10, "BTC/USDT", {
    'openType': 1, # isolated, 2 = cross
    'positionType': 1, # 1 = long, 2 = short
})
```
give error 
`ccxt.base.errors.BadSymbol: mexc does not have market symbol BTC/USDT`

**Comment by kroitor** [comment]
Pardon, @KindSpidey, I forgot – according to the info from the exchange admins, the BTC/ETH/LTC swap pairs are not tradable via the MEXC swap API yet. Try `ADA/USDT:USDT` to see if it makes any differences.

**Comment by KindSpidey** [comment]
Still `cxt.base.errors.ExchangeError: mexc {"success":false,"code":1001,"message":"Contract does not exist!"}`

**Comment by kroitor** [comment]
@KindSpidey please, paste your verbose output:

```Python
print('CCXT Version:', ccxt.__version__)
mexc_futures = ccxt.mexc({
    "apiKey": config.MEXC_API_KEY,
    "secret": config.MEXC_SECRET_KEY,
    "options": {'defaultType': 'swap' }
})
balance = mexc_futures.fetch_balance()["USDT"]["free"]
exchange.verbose = True
mexc_futures.set_leverage(10, "ADA/USDT:USDT", {
    'openType': 1, # isolated, 2 = cross
    'positionType': 1, # 1 = long, 2 = short
})
```

**Comment by kroitor** [comment]
@KindSpidey ok, that looks like a bug in CCXT, a wrong symbol is submitted, I've pushed a fix to it in CCXT 1.82.81 (the new build will arrive in ~25 minutes). Please, let me know if you have this same issue after upgrading to the most recent version 1.82.21 in ~25 minutes. Thanks for pointing it out!

**Comment by KindSpidey** [comment]
Now works
Thanks
